src/uct/api/uct.h

```diff
@@ -417,6 +417,10 @@ typedef enum uct_atomic_op {
 #define UCT_IFACE_FLAG_TAG_EAGER_BCOPY UCS_BIT(51) /**< Hardware tag matching bcopy eager support */
 #define UCT_IFACE_FLAG_TAG_EAGER_ZCOPY UCS_BIT(52) /**< Hardware tag matching zcopy eager support */
 #define UCT_IFACE_FLAG_TAG_RNDV_ZCOPY  UCS_BIT(53) /**< Hardware tag matching rendezvous zcopy support */
+
+#define UCT_IFACE_FLAG_FLUSH_CHECK     UCS_BIT(54) /**< Flush check operation */
```

Comment:
we already have `UCT_IFACE_FLAG_EP_CHECK`, which seems to be unused. Looks like it was intended for keepalive, 
@evgeny-leksikov plz clarify

Reply:
I can't be 100% sure bcz it was added earlier but looks like you are right

Reply:
leave few bits reserved - start from bit 60